doc/source/operations/control-plane-operation.rst

```diff
@@ -0,0 +1,424 @@
+-------------
+
+Follow the order in :ref:`full-shutdown`, but in reverse order.
+
+Shutting Down / Restarting Monitoring Services
+----------------------------------------------
+
+Shutting down
++++++++++++++
+
+Log into the monitoring host(s):
+
+.. code-block:: console
+
+   kayobe# ssh stack@monitoring0
+
+Stop all Docker containers:
+
+.. code-block:: console
+
+   monitoring0# for i in `docker ps -q`; do docker stop $i; done
+
+Shut down the node:
+
+.. code-block:: console
+
+   monitoring0# sudo shutdown -h
+
+Starting up
++++++++++++
+
+The monitoring services containers will automatically start when the monitoring
+node is powered back on.
+
+Software Updates
+================
+
+Sync local Pulp server with StackHPC Release Train
+--------------------------------------------------
+
+The host packages and Kolla container images are distributed from `StackHPC Release Train
+<https://stackhpc.github.io/stackhpc-release-train/>`__ to ensure tested and reliable
+software releases are provided.
+
+Syncing new StackHPC Release Train contents to local Pulp server is needed before updating
+host packages and/or Kolla services.
+
+To sync host packages:
+
+.. code-block:: console
+
+   kayobe# kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/pulp-repo-sync.yml
+   kayobe# kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/pulp-repo-publish.yml
+
+If the system is production environment and want to use packages tested in test/staging
+environment, you can promote them by:
+
+.. code-block:: console
+
+   kayobe# kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/pulp-repo-promote-production.yml
+
+To sync container images:
+
+.. code-block:: console
+
+   kayobe# kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/pulp-container-sync.yml
+   kayobe# kayobe playbook run $KAYOBE_CONFIG_PATH/ansible/pulp-container-publish.yml
+
+For more information about StackHPC Release Train, see :ref:`stackhpc-release-train` documentation.
+
+Once sync with StackHPC Release Train is done, new contents will be accessible from local
+Pulp server.
+
+Update Host Packages on Control Plane
+-------------------------------------
+
+Host packages can be updated with:
+
+.. code-block:: console
+
+   kayobe# kayobe overcloud host package update --limit <node> --packages '*'
+   kayobe# kayobe seed host package update --packages '*'
+
+See https://docs.openstack.org/kayobe/latest/administration/overcloud.html#updating-packages
+
+Upgrading OpenStack Services
+----------------------------
+
+* Update tags for the images in ``etc/kayobe/kolla-image-tags.yml``
+* Pull container images to overcloud hosts with ``kayobe overcloud container image pull``
+* Run ``kayobe overcloud service upgrade``
+
+You can update the subset of containers or hosts by
+
+.. code-block:: console
+
+   kayobe# kayobe overcloud service upgrade --kolla-tags <service> --limit <hostname> --kolla-limit <hostname>
+
+For more information, see: https://docs.openstack.org/kayobe/latest/upgrading.html
```

Comment:
I'd probably just delete this section. Full version upgrades require a lot more than this. For container tag bumps, we just use service deploy. Either way, I don't think this section is really that useful